MinimedKit/PumpEvents/PlaceholderPumpEvent.swift

```diff
@@ -9,26 +9,33 @@
 import Foundation
 
 
-public struct UnknownPumpEvent: TimestampedPumpEvent {
+public struct PlaceholderPumpEvent: TimestampedPumpEvent {
     public let length: Int
     public let rawData: NSData
     public let timestamp: NSDateComponents
+    public let name: String
```

Comment:
Ha! Well feel free to not listen to me. This case felt unique because it was storing the Type name as a string in itself. 

Now that we're keeping the raw bytes around, we could move them to computed properties, but we'd also have to keep the PumpModel around too.